lib/websocket/websocket-handler.ts

```diff
@@ -40,7 +40,6 @@ function getCorrelationIds(subscriptions: Subscription[]): number[] {
 function onConnect(socket: Interface.ISocket): void
 {
     initialize(socket)
```

Comment:
Since we remove the async setup part, the `initialize` doesn't have to return a promise anymore. I think a simple try/catch block here should be sufficient.